data/sql/updates/pending_db_world/netherwebvictim.sql

```diff
@@ -0,0 +1 @@
+UPDATE `creature_template` SET `unit_flags`=`unit_flags`|4|131072 WHERE `entry` = 22355;
```

Comment:
Use creature_template_movement instead of DISABLE_MOVE. Set rooted and ground both to 1.
You can change it in Keira

Reply:
whats the difference

Reply:
Honestly idk, I just heard it's preferred to use creature_template_movement as it interacts directly with the movement generators

Reply:
unit_flags are sniffed